management/dns_update.py

```diff
@@ -303,6 +304,32 @@ def has_rec(qname, rtype, prefix=None):
 		if not has_rec(qname, rtype):
 			records.append((qname, rtype, value, explanation))
 
+	# Adds autoconfiguration A records for all domains.
+	# mta-sts.* - required A record for mta-sts (serving the policy)
```

Comment:
We should be a little clearer in the comment here because MTA STS is a bit confusing, so e.g. for the comment:

> If this is a domain name that there are email addresses configured for, i.e. "something@" this domain name, then the domain name is a MTA-STS (https://tools.ietf.org/html/rfc8461) Policy Domain. A "_mta-sts" TXT record signals the presence of a MTA-STS policy, and an effectively random policy ID is used to signal that a new policy may (or may not) be deployed any time the DNS is updated. The policy itself is served at the "mta-sts" (no underscore) subdomain over HTTPS. The TLS certificate used by Postfix for STARTTLS must be a valid certificate for the MX name (PRIMARY_HOSTNAME), so we do not set an MTA-STS policy if the certificate is not valid (e.g. because it is self-signed and a valid certificate has not yet been provisioned).

Reply:
I added this comment and replaced line 307/308